book/forms.md

```diff
@@ -360,16 +360,18 @@ sheets[^update-styles] every time you type!
     area definitely can't make such changes, so let's skip this in our
     browser.
     
-Now when we click an `input` element and clear its contents, we can
-call `render` to redraw the page with the `input` cleared:
+Now when we click an `input` element and clear its contents, we should
+call `render` to redraw the page with the `input` cleared. To make things
+easier, I'm just always calling `render` regardless of whether things changed:
```

Comment:
```suggestion
call `render` to redraw the page with the `input` cleared. We also need
to call `render` if we clicked off an input element, since we might have unfocused
an input element in the process:
```